### ISSUE TITLE
File in FileShared may not have the "url" field

### ISSUE THREAD

**Issue by kiyoto** [slack-rs/slack-rs-api#14]
Here's a redacted example:

```js
{
    "type": "file_shared",
    "file": {
        "id": "XXX",
        "created": 1460414618,
        "timestamp": 1460414618,
        "name": "XXX",
        "title": "XXX",
        "mimetype": "XXX",
        "filetype": "XXX",
        "pretty_type": "XXX",
        "user": "XXX",
        "editable": false,
        "size": 733655,
        "mode": "XXX",
        "is_external": false,
        "external_type": "",
        "is_public": false,
        "public_url_shared": false,
        "display_as_bot": false,
        "username": "",
        "url_private": "XXX",
        "url_private_download": "XXX",
        "thumb_64": "XXX",
        "thumb_80": "XXX",
        "thumb_360": "XXX",
        "thumb_360_w": 360,
        "thumb_360_h": 220,
        "thumb_480": "XXX",
        "thumb_480_w": 480,
        "thumb_480_h": 294,
        "thumb_160": "XXX",
        "thumb_720": "XXX",
        "thumb_720_w": 720,
        "thumb_720_h": 441,
        "thumb_960": "XXX",
        "thumb_960_w": 960,
        "thumb_960_h": 587,
        "thumb_1024": "XXX",
        "thumb_1024_w": 1024,
        "thumb_1024_h": 627,
        "image_exif_rotation": 1,
        "original_w": 2850,
        "original_h": 1744,
        "permalink": "XXX",
        "permalink_public": "XXX",
        "channels": [],
        "groups": [],
        "ims": ["XXX"],
        "comments_count": 0
    },
    "event_ts": "1460414624.812120"
}
```

**Comment by kiyoto** [comment]
@mthjones ^

**Comment by mthjones** [comment]
Closed by #17.
